src/js/routes/VoterGuide/OrganizationVoterGuideEdit.jsx

```diff
@@ -101,7 +101,7 @@ export default class OrganizationVoterGuideEdit extends Component {
     const voter = VoterStore.getVoter();
     const voterGuideSaveResults = VoterGuideStore.getVoterGuideSaveResults();
     // console.log('onVoterGuideStoreChange voterGuideSaveResults:', voterGuideSaveResults);
-    if (voterGuideSaveResults && voter && voterGuideSaveResults.organization_we_vote_id === voter.linked_organization_we_vote_id) {
+    if (voterGuideSaveResults && voter && voterGuideSaveResults.organization_we_vote_id !== voter.linked_organization_we_vote_id) {
```

Comment:
Hi @SailingSteve why is this change necessary? I worry it changes the logic incorrectly.

Reply:
What was happening is that if `voterGuideSaveResults.organization_we_vote_id` was equal to `voter.linked_organization_we_vote_id` you would goToVoterGuideForDifferentElection with the same we_vote_id, you came back to line 104, and did the compare again, and then went right back to line 104, where they still were equal.  It was an endless loop, that locked up and crashed the session.

Reply:
Hi @SailingSteve, I just merged into develop a fix for the infinite loop and made a couple of other improvements to that component.